src/test/test-local-addresses.c

```diff
@@ -9,12 +9,10 @@
 #include "tests.h"
 
 static void print_local_addresses(struct local_address *a, unsigned n) {
-        for (unsigned i = 0; i < n; i++) {
-                _cleanup_free_ char *b = NULL;
-
-                assert_se(in_addr_to_string(a[i].family, &a[i].address, &b) >= 0);
-                log_debug("%s if%i scope=%i metric=%u address=%s", af_to_name(a[i].family), a[i].ifindex, a[i].scope, a[i].metric, b);
-        }
+        for (unsigned i = 0; i < n; i++)
```

Comment:
(should probably switch to size_t btw, as this just counts memory object. similar the func param above)